Neos.Flow/Classes/Persistence/Doctrine/AllowedObjectsListener.php

```diff
@@ -99,14 +100,27 @@ public function onFlush(OnFlushEventArgs $args)
         }
     }
 
+    /**
+     * Execute a dummy query on the given connection and return false if
```

Comment:
Wait I moved it here? This is so confusing, I think the change I took over had it here, SORRY! in that case I will have a look again why it was moved to this place...

Reply:
Right, looking through the history of this change, https://github.com/neos/flow-development-collection/pull/2637/commits/b17b9f0333499e2299700d4007b9f0d69312887c added it to the AllowedObjectsListener because it didn'T exist on the connection anymore!?